indico/web/client/js/react/components/Datepicker.jsx

```diff
@@ -0,0 +1,78 @@
+// This file is part of Indico.
```

Comment:
The lack of a border around the whole thing makes it a bit hard to see, especially on a page with mostly white background.